src/ApplicationInterface.php

```diff
@@ -1,11 +1,13 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Adding `declare(strict_types=1)` is a bit too risky on this lib for a minor release - can you please revert and disable the relevant rule for `src` only - having strict types enabled in `/test` is good and desirable.